guide.md

```diff
@@ -39,32 +39,30 @@ Illustration sources
 
 # The bitcoin design guide
 
-**Bitcoin is an incredibly open and public technology that we can all use, be part of, and contribute to. It is also a complex technology that takes time to understand, with a global audience and diverse uses cases. Therefore it's best to combine efforts, share knowledge and support each other in building great Bitcoin products.**
+**Bitcoin is a global payment protocol that anyone with Internet access can participate in and contribute to. However, because bitcoin’s use cases are as vast as its user base, building bitcoin applications can be complicated.**
 
-This guide was built on that very idea. Like a travel guide, it provides an introduction to a “foreign place”, shows you the highlights and tells you some of the best stories. It provides facts and maps to figure out how to get around, ideas for tours and things to do. Whether you end up taking the official museum tour, clubbing all night or go on shopping sprees is still your choice. Note that these are guides, not mandates. Just like in a foreign city, you can choose to go off the path, and forge a new direction. Others may just follow in the future.
+We designed this guide to help anyone navigate those complexities—regardless of where they live or work. We can’t imagine every possible use case, but we can help you build for them.
```

Comment:
The travel guide metaphor is visualized in the page header. So my vote would be for keeping it in the text as well. I also like the new copy, maybe combine the two?

Reply:
I think the "map" is evident from it being called a guide and a map being at the top. Otherwise it's see/say. I have some doubts about the metaphor itself and suspect that it's clearer without.

Reply:
What do you not like about the metaphor? It got a really strong positive response from people, which was a signal for me to put it in focus.

Reply:
I liked the metaphor when we were describing the 'guide project'.
No that I read/review the whole guide it feels we can be shorter in our writing in general.
Let people get to the content instead of describing what they will get. So I'm ok with moving on from the metaphor.

Reply:
I tend to agree, I was the biggest fan of the metaphor but after reading a guide a bit myself, I completely agree with what Daniel is saying above. We need to be a bit more concise. How about we remove it, and if we feel nostalgic we can always return it or we can put it on GitHub somewhere, repo description or something?

Reply:
Consensus is to remove it, so let's go with that. Thanks for the input.